homeassistant/components/systemmonitor/sensor.py

```diff
@@ -56,8 +64,9 @@
         "mdi:server-network",
         None,
     ],
-    "process": ["Process", " ", "mdi:memory", None],
-    "processor_use": ["Processor use", UNIT_PERCENTAGE, "mdi:memory", None],
+    "process": ["Process", " ", CPU_ICON, None],
+    "processor_use": ["Processor use (percent)", UNIT_PERCENTAGE, CPU_ICON, None],
```

Comment:
It would be good to wait with changing the name at least one release so the unique id is released first. That would make the change less breaking.

Reply:
Possibly split this PR so we add unique separately and release in 0.115. Then release the rest of these changes in 0.116.

Reply:
@MartinHjelmare To prevent that problem, I had originally set the entity_id directly, rather than just the unique_id. Since however the unique_id does not affect the entity_id at all and we cannot make it backwards-compatible either way in one release, I am even thinking about changing "processor_use" to "processor_use_percent" to be consistent with the naming of the other entities in this integration.

Do you think that naming consistency is worth the small break?

Reply:
Setting a unique id means we can change the entity id template, ie the name, without affecting existing users' entity_ids or custom names. The entity registry will apply the existing entity id when it finds an entity in the entity registry.